QA/acceptance-testing/README.md

```diff
@@ -0,0 +1,32 @@
+# Acceptance Testing
+
+## Why Acceptance Tests are Important
+
+- Writing the Acceptance Test lets the dev review their work. It can be very helpful in confirming that features are complete and that all edge cases are handled when describing what to do to a tester.
+- Acceptance testing helps find problems that the dev might not have experienced: issues coming from different hardware/environments, or a tester performing an action in a different way than the devs. Developers can get too close to their work and not approach their product the same way an end user does.
+- Acceptance testing saves time and frustration. It is better to find issues from someone who can explain and demonstrate how they caused it, rather than from an upset client. It's also easier to fix an issue earlier in the development timeframe than later when potentially more functionality is built on top of the problem.
+
+## How to create an Acceptance Test
+
+- Setup info - the tester needs to know where and how to perform a test. They will need a website URL/instructions on what to download, instructions on how to make an account or be provided credentials. They also need to know if they have to test with specific hardware, software, web browsers, etc.
+- A list of steps to perform the tests. This can be centralized in a primary GitHub issue with the steps given in a codeblock showing raw markdown that the tester can paste into their own issue. This allows the dev to write the steps in one spot, and each tester can create their own copy.
+- The steps given by the dev should cover the minimum amount of functionality that the tester needs to test. Testers should try to find issues and bugs beyond the listed instructions. The instructions are a starting point and give the tester an idea of how the feature should work.
+- The QA person will create their own issue, blank to start, which they will not assign.
+- The dev will assign the issue back to the QA tester with a link to the central issue where they can get the tests and begin testing.
```

Comment:
These last 2 steps don't make sense to me. Why do we need to link to the issue instead of assigning it to them?

Reply:
If I create an issue for someone who does not have write access to my repo, they cannot edit my issue. They can always edit their own issues, and I, as an administrator, can also.